README.md

```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
 
 <h2 align="center">Contributing</h2>
 
-Most of the time, if webpack is not working correctly for you it is a simple configuration issue.
+**We want contributing to webpack to be fun, enjoyable, and educational for anyone, and everyone.** We have a [vibrant ecosystem](https://medium.com/webpack/contributors-guide/home) that spans beyond this single repo. We welcome you to check out any of the repositories in [our](http://github.com/webpack) organization or [webpack-contrib](http://github.com/webpack-contrib) which houses all of our loaders and plugins.
 
-If you are still having difficulty after looking over your configuration carefully, please post
-a question to [StackOverflow with the webpack tag](https://stackoverflow.com/tags/webpack). Questions
-that include your webpack.config.js and relevant files are more likely to receive responses.
+Contributions go far beyond pull requests and commits. Although we love giving you the opportunity to put your stamp on webpack, we also are thrilled to receive a variety of other contributions including:
 
-If you have discovered a bug or have a feature suggestion, feel free to create an issue on Github.
+* [Documentation](https://github.com/webpack/webpack.js.org) updates, enhancements, designs, or bugfixes
 
-If you create a loader or plugin, please consider open sourcing it, putting it
-on npm and following the `x-loader`, `x-webpack-plugin` convention.
+* Spelling or grammar fixes
 
-You are also welcome to correct any spelling mistakes or any language issues.
+* README.md corrections or redesigns
 
-If you want to discuss something or just need help, [here is our Gitter room](https://gitter.im/webpack/webpack).
+* Adding unit, or functional tests
+
+* Triaging GitHub issues (especially helping find out if an issue still persists or is reproducible)
+
+* [Searching #webpack on twitter](https://twitter.com/search?q=webpack) and helping someone else who needs help
+
+* Teaching others how to contribute to one of the many webpack repo's!
+
+* [Blogging, speaking about, or creating tutorials](https://github.com/webpack-contrib/awesome-webpack) about one of webpack's many features. 
+
```

Comment:
This is somewhat nitpicky but you may want to remove the line breaks between list items to stay consistent with the rest of the doc (also I'm not sure all md processors will handle this as one list).